AGW-4969: Update header removal policy doc. (#398)
mulesoft/docs-api-manager: modules/ROOT/pages/header-removal-policy.adoc

Because every header that matches a Header Removal policy parameter is removed from the message, you must be careful about how you configure that parameter. 

> For example, the following parameter configuration removes the `Access-Control-Allow-Credentials` header from a message but also any other starting with `Access-Control` (like `Access-Control-Allow-Origin`). It also removes the `Cache-Control` header:

Reviewer: ```suggestion
For example, the following parameter configuration removes any header starting with the word `Access-Control`, such as the `Access-Control-Allow-Credentials` and `Access-Control-Allow-Origin` headers from a message. The configuration also removes the `Cache-Control` header:
```
